Mathlib/Topology/Instances/AddCircle.lean

```diff
@@ -505,6 +506,11 @@ end LinearOrderedField
 
 variable (p : ℝ)
 
+instance pathConnectedSpace : PathConnectedSpace $ AddCircle p := by
```

Comment:
I think that we should have a theorem `Function.Surjective.pathConnectedSpace` and an instance `PathConnectedSpace (Quotient _)`.

Reply:
Good idea, done.